lms/djangoapps/mobile_api/users/views.py

```diff
@@ -221,6 +221,7 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           * video_outline: The URI to get the list of all vides the user can
             access in the course.
           * id: The unique ID of the course.
+          * channel_id: A unique "clean" (alphanumeric with '_') ID of the course.
```

Comment:
Yeah, I was wondering about this as well.  Calling it something like "clean_id" would make it clearer about what exactly the content of the ID is.  However, calling it "channel_id" makes it clearer on the usage of that field.  
I wanted to make sure that all clients who want to subscribe to the course via push-notifications, would know that this is the field to use.  What if I call it "subscription_id" and update the description to include its expected usage?